public/app/features/alerting/unified/components/Label.tsx

```diff
@@ -13,15 +13,18 @@ interface Props {
   value: ReactNode;
   color?: string;
   size?: LabelSize;
+  onLabelClick?: (label: string, value: string) => void;
```

Comment:
nit: prefer the regular `onClick` naming convention here since we're applying it to the wrapper component so it functions like any other HTML element that supports `onClick`.